tests/pgsql/pgsql-bug-6983/test.rules

```diff
@@ -0,0 +1 @@
+alert pgsql any any -> any any (msg:"PGSQL Test Rule"; pkt_data; content:"|58|"; sid:1; rev:1;)
```

Comment:
I'll try that out :D

Reply:
While this can be done, due to what I described in the README (cf https://github.com/OISF/suricata-verify/pull/1796/commits/b5f8b39bceec64303047ad4215f072c8c3176c68#diff-bfd3f970befd93379e7a0cfb2bbee63fe0dd769725f6c5f35f04771bcfeab9afR5-R8), if we use a rule that matches on the contents of the select message, Suri doesn't log the transaction that triggered the alert, but some following, leading to no `pgsql` tx found to be logged. That's why I ended up using something short to match on - to at least have some match with the `pgsql` metadata logged.

Reply:
🤔 looks strange indeed

Reply:
Currently trying to debug this...